cmd/client.go

```diff
@@ -92,14 +95,22 @@ func (cli *Client) GetJobSpecs(c *clipkg.Context) error {
 func (cli *Client) CreateJobSpec(c *clipkg.Context) error {
 	cfg := cli.Config
 	if !c.Args().Present() {
-		return cli.errorOut(errors.New("Must pass in job JSON"))
+		return cli.errorOut(errors.New("Must pass in JSON or filepath"))
+	}
+	arg := c.Args().First()
+	var buf *bytes.Buffer
+	var err error
+	if gjson.Valid(arg) {
+		buf = bytes.NewBufferString(arg)
+	} else if buf, err = fromFile(arg); err != nil {
+		return cli.errorOut(err)
```

Comment:
In the event someone passes malformed json, they'll get back an error saying unable to open file. The prevent any misdirection or confusion, let's do the following for now:

```
return cli.errorOut(multierr.Append(errors.New("Must pass in valid JSON or filepath"), err))
```

We can DRY up that error in a var at the top of the file.